src/DynamoCoreWpf/Controls/NotificationsControl.xaml.cs

```diff
@@ -40,9 +40,9 @@ void NotificationsControl_Loaded(object sender, RoutedEventArgs e)
         /// <param name="e"></param>
         void window_PreviewMouseDown(object sender, MouseButtonEventArgs e)
         {
-            var hsvm = (HomeWorkspaceViewModel)((DynamoViewModel)DataContext).HomeSpaceViewModel;
+            var hsvm = (HomeWorkspaceViewModel)((DynamoViewModel)DataContext).HomeSpaceViewModel;            
             // Commented this after MAGN - 8423
-            // hsvm.ClearWarning();
+            //hsvm.ClearWarning();
```

Comment:
Would you revert the change to this file? In general, there should be a space after the `//` for comments

Reply:
Done